packages/cadl-ranch-specs/http/clients/parameters/path/main.cadl

```diff
@@ -0,0 +1,56 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/cadl-dpg";
+import "@cadl-lang/versioning";
+import "@azure-tools/cadl-azure-core";
+
+using Cadl.Http;
+using Azure.DPG;
+using Cadl.Versioning;
+using Azure.Core;
+using Cadl.Rest;
+
+@versioned(Versions)
+@service({
+  title: "PathClient",
+})
+@server(
+  "{Endpoint}/clients/parameters/path/{ApiVersion}",
+  "Test path parameters to client.",
+  {
+    @doc("""
+Pass in http://localhost:3000 for endpoint.
+""")
+    Endpoint: string,
+
+    @doc("Api Version")
+    @doc("""
+Pass in v1.0 for endpoint.
+""")
+    @path
+    ApiVersion: Versions,
+  }
+)
+namespace Clients.Parameters.Path;
+
+enum Versions {
+  v1_0: "v1.0",
+}
+
+@doc("Product resource")
+@resource("products")
+model Product {
+  @doc("ID of a product")
+  @key
+  id: int32;
+}
+
+@scenario
+@scenarioDoc("""
+Expected response body:
+  ```json
+  {"id": 1}
+  ```
+""")
+@doc("Get a resource")
+op get is ResourceRead<Product>;
```

Comment:
I would just remove the resource part no? If we are testing server params maybe just 
```
op noOperationParams(): NoContentResponse;

// And maybe adding this one to test it can still add path param for operations
op withOperationPathParam(@path id: string): NoContentResponse;
```

like we have for authentication scenarios

Reply:
I can split this up more, I kept it with templating bc we have api version both in the path, and the template will return api version as aquery param (this is what's happening in anomaly detector). Maybe the two operations you've listed, and then another one with a template to test the duplicate api versions?